Intersect (Core)/Config/Options.cs

```diff
@@ -149,6 +149,8 @@ public class Options
         public static int TileWidth => Instance.MapOpts.TileWidth;
 
         public static int TileHeight => Instance.MapOpts.TileHeight;
+        
+        public static int TimeUntilMapCleanup => Instance.MapOpts.TimeUntilMapCleanup;
```

Comment:
We're going to get rid of top level options.
Can you remove this and refer to it through the appropriate means? (Can't recall if Options.Instance.SubCategory or Options.SubCategory was preferred, may need to read recent changes)

Reply:
Done